Python/Ask_How_To_Qus_using_Voice/how_to.py

```diff
@@ -0,0 +1,48 @@
+import pyttsx3      
+import speech_recognition as sr                     
+from pywikihow import search_wikihow                
+
+engine = pyttsx3.init()
+
+def fun_talk(audio):
+    engine.say(audio)
+    engine.runAndWait()
+
+def get_command():
+
+    rec = sr.Recognizer()
+    with sr.Microphone() as source:
+        print("Say something...")
+        rec.pause_threshold = 1
+
+        audio = rec.listen(source)
+
+        try:
+            print("Recognizing...")
+            query = rec.recognize_google(audio, language='en-in')
+            print(f"You said: {query}\n")
+
+        except Exception as e:
+            # print(e)
+            print("Say that again please...")
+            return "None"
+
+        return query
+
+
+if __name__ == '__main__':
+
+    #while True:
+    if 1:
```

Comment:
This logic is inappropriate 
if statement is always going to be true. If not required you can remove that statement

Reply:
it is just in place of while so that the project can run only once...
but I will remove the both then it will work fine